ceph-iscsi.spec

```diff
@@ -132,22 +133,26 @@ ln -s service %{buildroot}%{_sbindir}/rcrbd-target-api
 
 %post
 %if 0%{?fedora} || 0%{?rhel}
-/bin/systemctl --system daemon-reload &> /dev/null || :
-/bin/systemctl --system enable rbd-target-gw &> /dev/null || :
-/bin/systemctl --system enable rbd-target-api &> /dev/null || :
+%systemd_post rbd-target-gw.service
+%systemd_post rbd-target-api.service
 %endif
 %if 0%{?suse_version}
 %service_add_post rbd-target-gw.service rbd-target-api.service
 %endif
 
 %preun
+%if 0%{?fedora} || 0%{?rhel}
+%systemd_prerun rbd-target-gw.service
+%systemd_prerun rbd-target-api.service
```

Comment:
Should be `%systemd_preun` (not "prerun") here I think

Reply:
You are right. Thanks. Fixed and repushed.